content/_data/changelogs/weekly.yml

```
@@ -11146,6 +11146,65 @@
   date: 2021-05-25
   changes: []
 
+- version: "2.296"
+  date: 2021-06-01
+  changes:
+    - type: major bug
+      category: regression
+      pull: 5479
+      issue: 65585
+      authors:
+        - thomasgl-orange
+        - basil
+      message: |-
+        Fix regressions in forms submissions from unwanted form validation in any browser (regression in 2.289).
+    - type: rfe
+      category: rfe
+      pull: 5472
+      authors:
+        - TobiX
+      message: |-
+        Show implied plugin dependencies or a count of dependencies for plugins split from core.
+    - type: bug
+      category: bug
+      pull: 5536
+      authors:
+        - Wadeck
+      message: |-
+        Improve contrast for the checkbox in the login page.
+    - type: bug
+      category: bug
+      pull: 5461
+      issue: 64991
+      authors:
+        - daniel-beck
+      message: |-
+        Jenkins redirects users to the previous page after login even if they were able to view it while not logged in (regression in 2.266).
+    - type: rfe
+      category: developer
+      pull: 5521
+      authors:
+        - basil
+      references:
+        - pull: 5521
+        - url: https://plugins.jenkins.io/nis-notification-lamp/
+          title: NIS notification lamp plugin
+      message: |-
+        Developer: Remove JTidy dependency from Jenkins core.
+        Plugins that use JTidy functionality must be updated to explicitly declare a dependency on JTidy rather than relying on Jenkins core to provide this library.
+
+  # pull: 5505 (PR title: Bump spring-security-bom from 5.4.6 to 5.5.0)
```

I think that I may have made a mistake when I guided you to not include this in the changelog.  Updates to the Spring security framework have been 
```suggestion
  # pull: 5505 (PR title: Bump spring-security-bom from 5.4.6 to 5.5.0)
```

See line 11210 for an example of the entry that should be used for this update.  It would look something like this:

```
  - type: rfe
    category: rfe
    pull: 5505
    authors:
    - dependabot[bot]
    references:
      - pull: 5505
      - url: https://github.com/spring-projects/spring-security/releases/tag/5.5.0
        title: Spring project spring-security 5.5.0 release notes
    message: |-
      Bump spring-security-bom from 5.4.6 to 5.5.0.
```

Could you include that in the pull request?

---

Definitely! Thanks again!